src/NearOrg/Notifications/Notification.jsx

```diff
@@ -14,12 +14,15 @@ const Notification = styled.div`
   }
 `;
 
-const Content = styled.div`
+const Content = styled.a`
```

Comment:
```suggestion
const Content = styled("Link")`
```

@erditkurteshiSQA check out this [discussion](https://github.com/near/near-discovery-components/discussions/498) for the reference

Reply:
@shelegdmitriy This has been addressed in the latest commits. Thank you for the review.